data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg

```diff
@@ -409,18 +415,14 @@
 
                 [message]
                     speaker=Minister Mefel
-                    message= _ "We have heard your summons, and we shall send these dark foes back to the abyss that spawned them!"
-                [/message]
-                [message]
-                    speaker=Deoran
-                    message= _ "Council of Westin, we are cheered by your presence. Your aid will be most welcome."
+                    message= _ "We heard your call for help and came as quickly as we could. These undead shall be sent back to the dark abyss that spawned them!"
```

Comment:
THAT dark abyss (as opposed to THIS dark abyss) is the one WHICH spawned the undead and to which we will be returning them.